src/Passes/examples/OptimisationUsingOpt/SimpleExample/Makefile

```diff
@@ -0,0 +1,5 @@
+clean:
```

Comment:
why are we only cleaning in make?

Reply:
There is no good reason for that. I've added a target for the QIR generation.